docs/user-guide.md

```diff
@@ -252,6 +270,13 @@ policies:
     branch: main
     directory: gke-policies
   - local: ./my-policies
+outputs:
+  - file: output-file.json
+  - pubsub:
+      topic: testTopic
+  - cloudStorage:
+      bucket: bucket
+      path: path/to/write
```

Comment:
I love the documentation update.
I think that outputs are so significant feature that we can add a mention to the `README.md` as well.
i.e. small chapter in "Usage" section after "checking multiple clusters".